Runtimes/Core/CMakeLists.txt

```diff
@@ -225,3 +224,22 @@ endif()
 if(SwiftCore_ENABLE_CONCURRENCY)
   add_subdirectory(Concurrency)
 endif()
+
+# Inter-project install info
+export(EXPORT SwiftCoreTargets
+  FILE "cmake/SwiftCore/SwiftCoreTargets.cmake")
+install(EXPORT SwiftCoreTargets
+  DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/SwiftCore"
+  FILE "SwiftCoreTargets.cmake"
+  COMPONENT SwiftCore_cmake)
+include(CMakePackageConfigHelpers)
+configure_package_config_file("${CMAKE_CURRENT_SOURCE_DIR}/cmake/installer/SwiftCoreConfig.cmake.in"
```

Comment:
Can we use a different name than "installer"? We currently have the macOS and Windows installers, and this can be confusing as to how they are related.

Reply:
changed to `cmake/interface/SwiftCoreConfig.cmake.in`.